### PR TITLE
Add documentation for single pattern matching

### PR DESCRIPTION
**Pull request by elnabo**
see #351 

Some white spaces have died in the way.

### CODE REVIEW

File: HaxeManual/06-language-features.tex [hunk 1]
```diff
@@ -515,6 +515,23 @@ \subsection{Useless pattern checks}
 
 
 
+\subsection{Single pattern check}
+\label{lf-pattern-matching-single}
+
+The compiler provides the function \expr{match} to check if an enum value match a given pattern:
```
Comment (Simn) [inline HaxeManual/06-language-features.tex]: "matches"

File: HaxeManual/06-language-features.tex [hunk 2]
```diff
@@ -515,6 +515,23 @@ \subsection{Useless pattern checks}
...
+
+\haxe[firstline=90,lastline=95]{assets/PatternMatching.hx}
+
+As this function only test if the pattern is matched, guards and variable capture are unavailable.
```
Comment (Simn) [inline HaxeManual/06-language-features.tex]: "tests"

**Comment by Simn** [comment]
Thanks, looks good! Just two small grammar things.

**Comment by elnabo** [comment]
Indeed, I alway forget that third person s. Fixed

**Pull request merged**